examples/database-with-web-server-and-redis/index.ts

```diff
@@ -0,0 +1,130 @@
+import { Database, Project, Redis, Services } from '@studion/infra-code-blocks';
+import * as pulumi from '@pulumi/pulumi';
+import * as aws from '@pulumi/aws';
+import * as awsx from '@pulumi/awsx';
+
+const databaseConfig = new pulumi.Config('database');
+const username = databaseConfig.require('username');
+const password = databaseConfig.require('password');
+const dbName = databaseConfig.require('dbname');
+
+const databasePasswordSecret = createPasswordSecret('database', password);
+
+const webServerImage = createWebServerImage();
+
+const project: Project = new Project('database-project', {
+  services: [
+    {
+      type: 'DATABASE',
+      serviceName: 'database-example',
+      dbName: dbName,
+      username,
+      password,
+      applyImmediately: true,
+      skipFinalSnapshot: true,
+    },
+    {
+      type: 'REDIS',
+      serviceName: 'redis',
+      dbName: 'test-db',
+      region: 'us-east-1',
+    },
+    {
+      type: 'WEB_SERVER',
+      serviceName: 'web-server-example',
+      port: 3000,
+      image: webServerImage.imageUri,
+      desiredCount: 1,
+      size: 'small',
+      autoscaling: { enabled: false },
+      environment: (services: Services) => {
+        const db = services['database-example'] as Database;
+        const redis = services['redis'] as Redis;
+
+        const redisPort = redis.instance.port.apply(port => port.toString());
+
+        return [
+          {
+            name: 'DATABASE_USERNAME',
+            value: username,
+          },
+          {
+            name: 'DATABASE_HOST',
+            value: db.instance.address,
+          },
+          {
+            name: 'DATABASE_DBNAME',
+            value: dbName,
+          },
+          {
+            name: 'REDIS_PORT',
+            value: redisPort,
+          },
+          {
+            name: 'REDIS_ENDPOINT',
+            value: redis.instance.endpoint,
+          },
+          {
+            name: 'NODE_ENV',
+            value: 'production',
+          },
+        ];
+      },
+      secrets: (services: Services) => {
+        const redis = services['redis'] as Redis;
+
+        const redisPasswordSecret = redis.instance.password.apply(password =>
+          createPasswordSecret('redis', password),
+        );
+
+        return [
+          {
+            name: 'DATABASE_PASSWORD',
+            valueFrom: databasePasswordSecret.arn,
+          },
+          {
+            name: 'REDIS_PASSWORD',
+            valueFrom: redisPasswordSecret.arn,
+          },
+        ];
+      },
+    },
```

Comment:
```suggestion
      secrets: (services: Services) => {
        const redis = services['redis'] as Redis;
        
        return [
          {
            name: 'DATABASE_PASSWORD',
            valueFrom: databasePasswordSecret.arn,
          },
          {
            name: 'REDIS_PASSWORD',
            valueFrom: redis.passwordSecret.arn,
          },
        ];
      },
    },
```
`passwordSecret` is already available on the Redis resource.

Reply:
The same goes for the database -> `database.password.secret.arn`